src/cli/traceViewer/traceViewer.ts

```diff
@@ -137,19 +137,13 @@ export async function showTraceViewer(traceStorageDir: string | undefined, trace
   if (!fs.existsSync(tracePath))
     throw new Error(`${tracePath} does not exist`);
 
-  let files: string[];
-  if (fs.statSync(tracePath).isFile()) {
-    files = [tracePath];
-    if (!traceStorageDir) {
-      traceStorageDir = path.dirname(tracePath);
+  const files: string[] = fs.statSync(tracePath).isFile() ? [tracePath] : collectFiles(tracePath);
 
-      if (fs.existsSync(traceStorageDir + '/trace-resources'))
-        traceStorageDir = traceStorageDir + '/trace-resources';
-    }
-  } else {
-    files = collectFiles(tracePath);
-    if (!traceStorageDir)
-      traceStorageDir = tracePath;
+  if (!traceStorageDir) {
+    traceStorageDir = fs.statSync(tracePath).isFile() ? path.dirname(tracePath) : tracePath;
+
+    if (fs.existsSync(traceStorageDir + '/trace-resources'))
+      traceStorageDir = traceStorageDir + '/trace-resources';
```

Comment:
@dgozman What do you think about this? I'm not sure what's your stance on ternary operators.

Reply:
Looks good, thanks!